From 6701212e86398622a05b85d87391646bf0e81bfc Mon Sep 17 00:00:00 2001
From: Marc Zyngier <marc.zyngier@arm.com>
Date: Fri, 4 Sep 2015 17:50:11 +0100
Subject: [PATCH 4/4] arm64: dts: Drop linux,pci-probe-only from the Seattle
 DTS
Git-commit: 6701212e86398622a05b85d87391646bf0e81bfc
Patch-mainline: v4.4-rc1

The linux,pci-probe-only property mandates an argument to indicate whether
or not to engage the "probe-only" mode, but the Seattle DTS just provides a
naked property, which is illegal.

Also, it turns out that the board is perfectly happy without probe-only, so
let's drop this altogether.

Signed-off-by: Marc Zyngier <marc.zyngier@arm.com>
Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
Reviewed-by: Rob Herring <robh@kernel.org>
Acked-by: Suravee Suthikulpanit <Suravee.Suthikulpanit@amd.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>

---
 arch/arm64/boot/dts/amd/amd-overdrive.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amd/amd-overdrive.dts b/arch/arm64/boot/dts/amd/amd-overdrive.dts
index 564a3f7..128fa94 100644
--- a/arch/arm64/boot/dts/amd/amd-overdrive.dts
+++ b/arch/arm64/boot/dts/amd/amd-overdrive.dts
@@ -14,7 +14,6 @@
 
 	chosen {
 		stdout-path = &serial0;
-		linux,pci-probe-only;
 	};
 };
 
-- 
2.1.4

